languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml

```diff
@@ -15450,6 +15450,17 @@ USA
             <!--
       Eu acho que é perigoso subir aquela montanha sozinho. → Eu acho ser perigoso subir aquela montanha sozinho.
       -->
+            <antipattern>
+                <token>que</token>
+                <token>é</token>
+                <token postag='VMSF3.+' postag_regexp='yes'/>
```

Comment:
This postag is only for the 3sg future subjunctive. It will match on `que é *fizer`, but not `que é fazer`. You probably meant to use the infinitive tag, didn't you?

Reply:
ohhhh… right. The original idea was “fazer”. Give me a few minutes and I will fix it.